site/en/blog/avoiding-oversharing-when-screen-sharing/index.md

```diff
@@ -0,0 +1,100 @@
+
+When [`getDisplayMedia()`] is called, users are presented with a media picker which allows them to share whichever display surface they would like any tab, window, or screen. Experience shows that the ordering of the offering affects the users’ selection. We are experimenting in Chrome&nbsp;107 on desktop with reordering the picker in order to nudge users towards tabs, which is the best option in the majority of cases.
+
+<figure>
+  {% Img src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/7z4i2ZI77UZA80WRg4Rc.jpg", alt="Screenshots of the old and new media picker prompts.", width="800", height="556" %}
+  <figcaption>
+    Chrome’s media picker (old vs. new).
+  </figcaption>
+</figure>
+
+The expected benefits are as follows:
+
+### Privacy
+
+Previously, screens were offered as the most prominent option. This is the least privacy-preserving option, as users may end up exposing more personal information than they had originally intended. Such as:
+- Timezone (via the system clock)
+- Language
+- System notifications
+- Running apps
+- Installed apps
+- Installed extensions
+- Bookmarks
+- Browsing history (via the omnibox)
+- The user’s profile pictures (browser, system, other apps)
+- Rugby-club affiliation (via desktop wallpapers)
+- Audio notifications from backgrounded apps, like a calendar
+
+Most of these concerns are mitigated by sharing a tab instead of a screen.
+
+### Feature-completeness
+
+Tab sharing is more feature-complete:
+- Sharing tab audio is supported on all platforms; sharing system audio and window audio is only implemented on some platforms, at the time of writing.
+- When sharing tabs, complementary features enable closer collaboration with the captured content:
+  - [Capture Handle] allows the identification of the captured tab and establishment of a communications channel with it.
+  - [Region Capture] allows further scoping down of captured content.
+
+### Performance and User Experience
+
+When a user intends to share another web app, it’s preferable to directly share the tab running that app, rather than its Chrome window, or even the entire screen.
+- The tab will continue being shared even if the user interacts with another app.
+- The resulting image on remote users’ participants is going to be “zoomed in on the action” and therefore clearer.
+- Avoiding the capture, encoding and transmission of irrelevant content saves CPU and bandwidth.
+- A more stable frame-rate is possible for tab-capture than for other types of capture.
+
+## New screen sharing controls
+
+The new `displaySurface`, `surfaceSwitching`, `selfBrowserSurface`, and `systemAudio` options will allow you to tailor your screen sharing experience.
+
+Check out [Privacy-preserving screen sharing controls] for more information.
+
+## What's next {: #future }
+
+[Conditional Focus] will allow the capturing web app to instruct the browser to either switch focus to the captured display surface, or to avoid such a focus change.
```

Comment:
Shall we mention suppressLocalAudioPlayback as well, do you think?

Reply:
Sure.